lib/ACF/Layouts/CustomHtmlLayout.php

```diff
@@ -52,7 +52,7 @@ private function add_layout_fields() : void {
             ],
         ];
 
-        if ( user_can( get_current_user_id(), 'unfiltered_html' ) ) {
+        if ( \user_can( \get_current_user_id(), 'unfiltered_html' )  || ! \is_admin() ) {
```

Comment:
Lipsahtanu ylimääränen välilyönti
```suggestion
        if ( \user_can( \get_current_user_id(), 'unfiltered_html' ) || ! \is_admin() ) {
```